Plugins/RSBot.General/PacketHandler/CharacterListing.cs

```diff
@@ -126,6 +126,8 @@ public void Invoke(Packet packet)
 
             if (lobbyCharacters.Contains(selectedAccount.SelectedCharacter))
                 Components.AutoLogin.EnterGame(selectedAccount.SelectedCharacter);
+            else
+                Components.AutoLogin.EnterGame(lobbyCharacters[0]);
```

Comment:
If list empty you will get OutOfRange exception. Check list empty or not.

Reply:
I have changed everything here just no time to send